notebooks/261-softvc-voice-conversion/261-softvc-voice-conversion.ipynb

```diff
@@ -0,0 +1,440 @@
+{
```

Comment:
<p>maybe better to add directory in sys.path?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1384/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Reply:
It is better, but so-vits-svc project had some hardcoded paths, like `"pretrain/checkpoint_best_legacy_500.pt"` and there is no way to pass it without changing the code.